svrestarter.go

```diff
@@ -56,13 +59,21 @@ func (s *SvRestarter) Restart() error {
 		if err != nil {
 			if strings.Contains(string(out), "timeout: run: ") {
 				err = ErrRestartTimeout{Service: s.Service}
+				tags = append(tags, "status:timeout")
 			} else {
 				err = ErrRestartFailed{Service: s.Service, Message: string(out)}
+				tags = append(tags, "status:success")
 			}
 		}
 	case <-s.preempt:
 		<-preemptionAcceptable
 		err = ErrRestartPreempted{Service: s.Service}
+		tags = append(tags, "status:preempted")
+	}
+
+	if Statsd != nil {
```

Comment:
same deal here; you've asserted `Statsd` must be successfully configured, so don't bother with the conditional.